src/main/java/com/example/glovo/model/Order.java

```diff
@@ -1,4 +1,18 @@
 package com.example.glovo.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import java.util.List;
+
+@AllArgsConstructor
+@Data
+@Builder
 public class Order {
+    private long id;
+    private String customerName;
+    private String customerNumber;
```

Comment:
Why use this parameter if you already have a unique **id**?

Reply:
I mean phone number, not unique id. My bad with variable naming.